### PR TITLE
feat: add sitemap

### LINKED ISSUE #416: Create site sitemap

### PR DESCRIPTION
**Pull request by JustinCast**
# Add sitemap
This PR adds sitemap file to enhance SEO ranking
# Linked issues
closes #416

### CODE REVIEW

File: webapp/src/sitemap.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:news="http://www.google.com/schemas/sitemap-news/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:mobile="http://www.google.com/schemas/sitemap-mobile/1.0" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1" xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">
+  <url>
+    <loc>https://lifebank.io/</loc>
+    <changefreq>weekly</changefreq>
+    <priority>0.5</priority>
+  </url>
+  <url>
+    <loc>https://lifebank.io/not-found</loc>
```
Comment (xavier506) [inline webapp/src/sitemap.xml:8]: i dont think we need the 404 indexed

File: webapp/src/sitemap.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <changefreq>weekly</changefreq>
+    <priority>0.5</priority>
+  </url>
+  <url>
+    <loc>https://lifebank.io/not-found</loc>
```
Comment (xavier506) [inline webapp/src/sitemap.xml:13]: seems like a duplicate route

File: webapp/src/sitemap.xml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    <changefreq>yearly</changefreq>
+    <priority>0.5</priority>
+  </url>
+  <url>
+    <loc>https://lifebank.io/help</loc>
+    <changefreq>yearly</changefreq>
+    <priority>0.5</priority>
+  </url>
+  <url>
+    <loc>https://lifebank.io/internal-error</loc>
```
Comment (xavier506) [inline webapp/src/sitemap.xml:33]: no need for 500 errors to be indexed

**Review by JeanVegaD — APPROVED** [review]

**Review by AngeloCG97 — APPROVED** [review]

File: webapp/src/sitemap.xml (hunk 1 above)
Comment (JustinCast) [reply webapp/src/sitemap.xml:8]: Got it

**Pull request merged**